### ISSUE TITLE
only one camera works

### ISSUE THREAD

**Issue by laviniug** [zaneclaes/home-assistant-motion-eye-component#3]
The component seems to work fine and it detects all 4 cameras but only one works.

020-08-21 11:50:34 INFO (MainThread) [homeassistant.helpers.entity_registry] Registered new camera.motion_eye entity: camera.road
2020-08-21 11:50:34 INFO (MainThread) [homeassistant.helpers.entity_registry] Registered new camera.motion_eye entity: camera.yard_right
2020-08-21 11:50:34 INFO (MainThread) [homeassistant.helpers.entity_registry] Registered new camera.motion_eye entity: camera.conservatory
2020-08-21 11:50:34 INFO (MainThread) [homeassistant.helpers.entity_registry] Registered new camera.motion_eye entity: camera.yard_main

All other 3 cameras throw this type of error in the network tab:
	
GET
	https://home.gavanescu.com:8123/api/camera_proxy_stream/camera.yard_main?token=4f***
Status502
Bad Gateway
VersionHTTP/1.1
Transferred16 B (16 B size)

	
Content-Length
	16
Content-Type
	text/plain; charset=utf-8
Date
	Fri, 21 Aug 2020 08:52:44 GMT
Server
	Python/3.8 aiohttp/3.6.2

**Comment by zaneclaes** [comment]
Have a look at the camera config in HA. What is the mjpeg URL? Make sure you can load the streaming URL from your motion eye server. It sounds like HA can’t access the other 3 ports on your server. Since each camera uses a different port, it’s not uncommon to forget to open the other ports. If that’s the case, you’ll need to open the ports or otherwise ensure the streaming APIs are accessible.

**Comment by Chris-Johnston** [comment]
Thank you @zaneclaes ! I was having a similar issue, and this was the hint I needed. In my case, I had three cameras that all showed stills fine in HA, but only the first (on port 8081) would stream. On another machine I checked with `nmap 10.x.x.x -p 8080-8090` and saw that only port 8081 was open.

I run motioneye using docker-compose. My problem was that I only allowed ports 8765 and 8081.

```yaml
services:
  motioneye:
    ports:
      - "8081:8081" # only Camera 1 will work
      - "8765:8765"
```

The solution was to allow more ports:

```diff
-      - "8081:8081"
+      - "8081-8099:8081-8099"
```

I killed and restarted the motioneye container and it works now!
